index.js

```
@@ -185,23 +186,27 @@ embed.vimeo.image = function (id, opts, cb) {
 
   opts.image = validVimeoOpts.indexOf(opts.image) >= 0 ? opts.image : 'thumbnail_large'
 
-  request.get({
-    url: 'http://vimeo.com/api/v2/video/' + id + '.json',
-    json: true
-  }, function (err, res, body) {
-    if (err) return cb(err)
-    if (res.statusCode !== 200) return cb(new Error('unexpected response from vimeo'))
-    if (!body || !body[0] || !body[0][opts.image]) return cb(new Error('no image found for vimeo.com/' + id))
+  fetch('http://vimeo.com/api/v2/video/' + id + '.json')
+    .then(function (res) {
+      if (res.status !== 200) return cb(new Error('unexpected response from vimeo'))
```

Please throw this error so the catch block can call the callback